github/ci/prow-deploy/molecule/default/vars/kubevirtci-testing.yml

```diff
@@ -6,3 +6,7 @@ kubeconfig_path: '/root/.kube/config'
 secrets_dir: '{{ project_infra_root }}/github/ci/prow-deploy/kustom/overlays/kubevirtci-testing/secrets/'
 
 kind_cluster_config: '/workspace/cluster.yaml'
+
+SRIOVNodes:
+- name: node01
```

Comment:
can this be auto detected ? 
just in case we use the deploy with other provider, i.e kind-sriov
where the nodes have other name

Not crucial atm, up to you 
maybe can be also be a ENV var with a default value
so its easy to override

Reply:
> can this be auto detected ?

Not sure, can we determine from the node attributes if it should be patched? If so we would not need the static definition of vars at all.

> maybe can be also be a ENV var with a default value

But we need to define an array of nodes to be patched right? I would avoid env vars for that,let me know wdyt

Reply:
> > can this be auto detected ?
> 
> Not sure, can we determine from the node attributes if it should be patched? If so we would not need the static definition of vars at all.

Talking only on the smoke tests, and testing deploy, not about the IBM real one, which must have specific name + capacity tuples as you did.
Meant that we just need to patch one of the nodes, no matter which, for smoke tests.
If if someone will use the deploy script with other provider, this would fail, because the node name isn't node01.

> 
> > maybe can be also be a ENV var with a default value
> 
> But we need to define an array of nodes to be patched right? I would avoid env vars for that,let me know wdyt

On the IBM totally true, and there we dont need the above changes,
for the IBM you did totally fine, and it will have indeed specific node name + capacity,
as for example we have now srv17 - 4, srv20 - 2 and so on.

Reply:
Feel free to disregard, in case someone will update the provider in the deploy (for testing locally) to other provider, he will be able to remove the check or adapt it for the right node name, thanks

Reply:
I see what you mean, in the testing environment we are creating the test cluster and we are actually defining the node name (we need to label it in a specific way so that some workloads are scheduled to it). I've modified the code a bit to have the node name as a variable and use it both for naming the node and for defining the name of the SRIOV node, PTAL.